servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,68 @@
+
+Turn off [Audit Forwarding](/servicecontrol/errorlog-auditlog-behavior.md) if it is not needed. ServiceControl sends a copy of each audited message to configured Audit Forwarding queue. If these messages are not being used, turn this feature off.
+
+
+## Plugin Considerations
+
+[Plugins](/servicecontrol/plugins/) are installed in an Endpoint and send data to ServiceControl. This communication uses messaging over the configured transport of the endpoint. Each instance of a plugin adds more messages to the ServiceControl queue which can delay the processing of each message and make ServiceControl less responsive.
+
+
+### Heartbeats
+
+Not all endpoints are mission critical and need to be monitored with [heartbeats](/servicepulse/intro-endpoints-heartbeats.md) using the same SLA. For endpoints that are less critical to business, [adjust the Heartbeat Interval](/servicecontrol/plugins/heartbeat#configuration-heartbeat-interval). Increasing the interval ensures that ServiceControl is able to process heartbeats in a timely manner. 
+
+Heartbeat messages tend to be frequent, and a large backlog can occur if ServiceControl is offline for an extended period. When this happens, it can take ServiceControl some time to process old heartbeats when it restarts. 
+
+
+### Saga Audit
+
+The [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugin produces a lot of data. It's use outside of a development environment is not recommended. If the saga audit plugin is already deployed in production, [follow the instructions outlined](/servicecontrol/plugins/saga-audit#removing-the-plugin-from-production) to remove it.
+
+
+## Performance Considerations
+
+Run a performance test using the expected peak and average message throughput for the environment to baseline the system. This test suite should include simulating infrastructure outages that will cause large numbers of message processing failures. The baseline test should not include audit ingestion or any ServiceControl plugins. 
+
+Once this baseline has been established, follow these steps:
+- For each endpoint, install and configure the Heartbeat plugin (if needed). Re-run the performance test suite and monitor ServiceControl to ensure that it can effectively monitor the system under load. This may require adjustments to the Heartbeat interval. Re-run the performance tests after each adjustment.
+- For each endpoint, turn on auditing (if needed) and re-run the performance tests to assess impact.
+- For each endpoint, turn on any required Custom Checks and re-run the performance tests to assess impact.
+
+
+## Anti-Virus Checks
+
+Exclude the ServiceControl [database directory](/servicecontrol/configure-ravendb-location.md) from anti-virus checks. ServiceControl uses an embedded database and produces a lot of I/O traffic. Some anti-virus software can add overhead to I/O operations causing a significant performance impact.
```

Comment:
"a lot of I/O traffic" > "a high level of I/O traffic"

required picture of an alot http://thewritepractice.com/wp-content/uploads/2012/05/Alot-vs-a-lot1-600x450.png